### ISSUE TITLE
ap

### ISSUE THREAD

**Issue by ywqly** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#311]
May I ask how much the AP value has dropped? The article said that it has dropped by 1%. Why do I feel that it is not only the drop of these

**Comment by Daniil-Osokin** [comment]
Hi! The performance is consistent with the paper. Many works, when reporting accuracy in the leaderboard, use test-time augmentations, such as image flip or multi-scale testing. This work uses single network forward pass for testing by default, you can enable multi-scale testing by passing `--multiscale` key to the validation script.

**Comment by ywqly** [comment]
Thanks for your reply, I have another question, how can I get the visual test results in val.py

**Comment by Daniil-Osokin** [comment]
Pass [`--visualize`](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/val.py#L171) key to the script.

**Comment by ywqly** [comment]
Thank you very much for your reply again. May I ask what is included in this visualization result, and I have modified the network layer of your code and reached 38.8% in the first stage of training. Is this an improvement?

**Comment by Daniil-Osokin** [comment]
I suggest to follow the [README.md](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/README.md) file to check proper usage of the validation script.

**Comment by Daniil-Osokin** [comment]
Hope, it is helped.
